sources/Util/Utils.hpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by cbihan on 26/05/2021.
+//
+
+#pragma once
+
+#include "Vector/Vector3.hpp"
+#include "Models/Vector3.hpp"
+
+namespace BBM
+{
+	struct Utils
+	{
+		//! @brief Convert BBM Vector3f to RAY Vector3
+		static RAY::Vector3 toRAY(const BBM::Vector3f &wal);
```

Comment:
Don't need this anymore if we use the *operator

Reply:
fixed -> https://github.com/AnonymusRaccoon/Bomberman/pull/64/commits/6bd7a46ea978636174595209c16c3a0b4e942f56